clang/test/SemaCXX/coroutines.cpp

```diff
@@ -291,6 +294,38 @@ void mixed_coreturn_template2(bool b, T) {
     return; // expected-error {{not allowed in coroutine}}
 }
 
+struct promise_handle;
+
+struct Handle : std::coroutine_handle<promise_handle> { // expected-note 2{{candidate constructor (the implicit copy constructor) not viable}}
```

Comment:
NIT: for brevity we could use a shorter substring (same below)
```suggestion
struct Handle : std::coroutine_handle<promise_handle> { // expected-note 2{{not viable}}
```

Reply:
Done.